src/vs/workbench/contrib/chat/browser/chatDragAndDrop.ts

```diff
@@ -89,13 +101,14 @@ export class ChatDragAndDrop extends Themable {
 	}
 
 	private currentActiveTarget: HTMLElement | undefined = undefined;
-	private createOverlay(target: HTMLElement, overlayContainer: HTMLElement): { overlay: HTMLElement; disposable: IDisposable } {
+	private createOverlay(target: HTMLElement, overlayContainer: HTMLElement): { overlay: HTMLElement; disposable: MutableDisposable<IDisposable> } {
 		const overlay = document.createElement('div');
 		overlay.classList.add('chat-dnd-overlay');
 		this.updateOverlayStyles(overlay);
 		overlayContainer.appendChild(overlay);
 
-		const disposable = new DragAndDropObserver(target, {
+		const disposable = new MutableDisposable<IDisposable>();
+		disposable.value = new DragAndDropObserver(target, {
```

Comment:
No need to use a mutable disposable here since it's never changed?

Reply:
ah shoot yes leftover from when i tried to use mutable disposable at first